modules/service/src/main/scala/lucuma/odb/service/CalibrationConfigSubset.scala

```diff
@@ -123,6 +123,7 @@ object CalibrationConfigSubset:
         GmosImagingInput.Create.North(
           filters,
           GmosImagingInput.Create.Common(
+            none, // Do we need multipleFiltersMode here?
```

Comment:
I see the similar comment for offsets.  I assume `none` means not explicit multiple filters mode which seems correct to me.

Reply:
Right, this means no explicit multiple filters mode. It is probably correct, but Carlos says that science hasn't yet specified the behavior.